{
  "severity": "LOW", 
  "created_at": "2016-04-08T15:59:00Z", 
  "description": "The password hasher in contrib/auth/hashers.py in Django before 1.8.10 and 1.9.x before 1.9.3 allows remote attackers to enumerate users via a timing attack involving login requests.", 
  "updated_at": "2017-09-08T01:29:00Z", 
  "packages": [
    {
      "vulnerable_version": "1.3.1-4ubuntu1.19", 
      "name": "python-django", 
      "source": "USN", 
      "fixed_version": "1.3.1-4ubuntu1.20", 
      "uid": null, 
      "fixed_package": {
        "source": "python-django", 
        "version": "1.3.1-4ubuntu1.20", 
        "name": "python-django-doc", 
        "sha256": "0fcb7207a82e91d53d29e73fcde61407220f8ca51f41cd7a15dd9d311e337879", 
        "sha1": "8f351a85640dd27b74878cdd9c8c837c4bf3b354", 
        "md5": "4e2175972815818dc543ff2ebd523834", 
        "arch": "ALL", 
        "filename": "python-django-doc_1.3.1-4ubuntu1.20_all.deb"
      }, 
      "vulnerable_package": {
        "source": "python-django", 
        "version": "1.3.1-4ubuntu1.19", 
        "name": "python-django-doc", 
        "sha256": "a0a32e9a509ed0605bb6a744a3332c73d9771d6cbaa570a590aac3fa179ed53d", 
        "sha1": "d1a3734b8e22b8509b85c57b7a8ef4fee3dd4205", 
        "md5": "7790b7dc2cd4a0c9ef22919205d47416", 
        "arch": "ALL", 
        "filename": "python-django-doc_1.3.1-4ubuntu1.19_all.deb"
      }
    }, 
    {
      "vulnerable_version": "1.3.1-4ubuntu1.19", 
      "name": "python-django", 
      "source": "USN", 
      "fixed_version": "1.3.1-4ubuntu1.20", 
      "uid": null, 
      "fixed_package": {
        "source": null, 
        "version": "1.3.1-4ubuntu1.20", 
        "name": "python-django", 
        "sha256": "5f57fc5bd590049598d2c509b30d509ce156222d0d7399a4ad19a5e7925b631b", 
        "sha1": "0d574e74903d5cdccc9d9e5b15d8424f38680bc7", 
        "md5": "cf5a69222b57c438f2e2ef105dbc2c31", 
        "arch": "ALL", 
        "filename": "python-django_1.3.1-4ubuntu1.20_all.deb"
      }, 
      "vulnerable_package": {
        "source": null, 
        "version": "1.3.1-4ubuntu1.19", 
        "name": "python-django", 
        "sha256": "3b04f3f21a32f425dfe213b0ba2a09a01590d2d720c9fbf987aee20e0606a651", 
        "sha1": "69ae73c2c8f0036a79a9d2d77feec2d8781285e3", 
        "md5": "b04d2b27e5c84536ff1a698761653018", 
        "arch": "ALL", 
        "filename": "python-django_1.3.1-4ubuntu1.19_all.deb"
      }
    }, 
    {
      "vulnerable_version": "1.6.1-2ubuntu0.11", 
      "name": "python-django", 
      "source": "USN", 
      "fixed_version": "1.6.1-2ubuntu0.12", 
      "uid": null, 
      "fixed_package": null, 
      "vulnerable_package": {
        "source": "python-django", 
        "version": "1.6.1-2ubuntu0.11", 
        "name": "python-django-doc", 
        "sha256": "e5f1cbdf264e426899bc421aefd3722c5310c50bd6c1b8b7ad4de72062286542", 
        "sha1": "c67c8b1ace055f3ce827df8fb4a669b21214da81", 
        "md5": "de505be01ca63f4754863e2634e54176", 
        "arch": "ALL", 
        "filename": "python-django-doc_1.6.1-2ubuntu0.11_all.deb"
      }
    }, 
    {
      "vulnerable_version": "1.6.1-2ubuntu0.11", 
      "name": "python-django", 
      "source": "USN", 
      "fixed_version": "1.6.1-2ubuntu0.12", 
      "uid": null, 
      "fixed_package": null, 
      "vulnerable_package": {
        "source": null, 
        "version": "1.6.1-2ubuntu0.11", 
        "name": "python-django", 
        "sha256": "0e22ebb570859d8a76161ab95214a63bd0ecaa7194dfd24bf1dd481085d4ae7b", 
        "sha1": "e846959ab4b84728cf8d12f6c740d4eeabdbbcb2", 
        "md5": "03d21f3932e5ab17e8c915041de7ad59", 
        "arch": "ALL", 
        "filename": "python-django_1.6.1-2ubuntu0.11_all.deb"
      }
    }
  ], 
  "name": "CVE-2016-2513"
}